packages/client/src/App.sass

```diff
@@ -213,6 +215,8 @@ h2
       border-bottom: 1px solid $grey-secondary
     &-right
       border-right: 1px solid $grey-secondary
+  &-yellow
```

Comment:
suggest renaming to warning to keep consistent with the rest of the names

Reply:
I would rather call it &-details or &-error, because the warning is orange, and error modal has error details
Or even &-alert, because I see it here in Figma that it's planned for toasts like this
https://www.figma.com/file/sKtcz6ytBmNNDNW7XU8ea5/Vessel%3A-User-Experience?node-id=312%3A84684
I'm bad with naming, what do you think is the best out of these three proposals?